expyre/cli/cli.py

```diff
@@ -41,8 +43,48 @@ def cli_ls(ctx, id, name, status, system):
         print(f"No matching jobs in JobsDB at {config.db.db_filename}")
     else:
         print("Jobs:")
+        # {'id': 'vasp_eval_chunk_0_KEiSzsC7ft8ASaOmurO4yw5PhlCYVLgpAcVxVXvc1e0=_1kxwmf_c',
+        # 'name': 'vasp_eval_chunk_0',
+        # 'from_dir': '/home/cluster2/bernstei/src/work/Perovskites/ACE/_expyre/run_vasp_eval_chunk_0_KEiSzsC7ft8ASaOmurO4yw5PhlCYVLgpAcVxVXvc1e0=_1kxwmf_c',
+        # 'status': 'started',
+        # 'system': 'onyx',
+        # 'remote_id': '1805770.pbs01',
+        # 'remote_status': 'running',
+        # 'creation_time': '2021-12-06 10:23:04',
+        # 'status_time': '2021-12-06 10:25:42'}
+
+        cols = []
         for job in jobs:
-            print(job)
+            headers = [] # ugly, but last loop iter is what counts
```

Comment:
I don't know if it's worth rewriting, but fyi - I've used pandas for printing nice tables before. For example, [this function](https://github.com/gelzinyte/scripties/blob/338ac6ebdfb39842632b3af30364df497515fc00/util/bde/table.py#L93) sets up and prints the table and [this one](https://github.com/gelzinyte/scripties/blob/338ac6ebdfb39842632b3af30364df497515fc00/util/bde/table.py#L301) adds an entry.

Reply:
I'll take a look, and see how simple it looks to use.

Reply:
Let me know what you think of the pandas output.

Reply:
There's a missing `f` in line 77 of `cli.py` (` rows['from dir'].append("{job['from_dir']}")`), but otherwise looks great, thanks for the cleanup.

Reply:
I knew I worked on some of these, but now this one, and I also didn't finish pushing.  Should be fixed in the commit I'll push in a few minutes.